code-challenges.test.js

```diff
@@ -22,6 +29,73 @@ const fibLength2 = 10
 
 // b) Create the function that makes the test pass.
 
+
+describe("fibonacci", () =>{
+  it("takes in a number greater than 2 and creates a new array", () => {
+    expect(fibonacci(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+  }
+)})
+
```

Comment:
To keep your code dry with MVP in mind, we can refactor your test and turn 2 tests into 1 that tests both criteria. It would be describe("fibonacci", () =>{
  it("takes in a number greater than 2 and creates a new array", () => {
    expect(fibonacci(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
expect(fibonacci(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
  }
)})

By keeping everything as you had it and including both of your expect statements.